syntax = "proto3";

package stream_server;

message RegisterStreamRequest {
  string stream_id = 1;
  int32 message_count = 2;  // Number of stream messages to send
}

message RegisterStreamResponse {
  bool success = 1;
  string message = 2;
}

service StreamServer {
  rpc RegisterStream(RegisterStreamRequest) returns (RegisterStreamResponse);
}
